FirebaseRemoteConfig/Sources/RCNDevice.m

```diff
@@ -104,6 +104,8 @@ RCNDeviceModel FIRRemoteConfigDeviceSubtype(void) {
     @"ms" : @[ @"ms_MY" ],
     // Maltese
     @"mt" : @[ @"mt_MT" ],
+    // Marathi
+       @"mr" : @[ @"mr_IN" ],
```

Comment:
Indenting two spaces less should fix the CI failure. See also https://github.com/firebase/firebase-ios-sdk/blob/main/CONTRIBUTING.md#styling-your-code

Reply:
Done. Thanks Paul.